README.md

```diff
@@ -22,10 +22,15 @@ custom:
   client:
     bucketName: unique-s3-bucketname-for-your-website-files
     distributionFolder: client/dist # (Optional) The location of your website. This defaults to client/dist
+    indexDocument: index.html # (Optional) The location of your index document. Defaults to index.html
+    errorDocument: error.html # (Optional) The location of your error document. Defaults to error.html
+    spa: false # (Optional) Set this for single-page applications to redirect all errors to the index document
 ```
 
 * **Warning:** The plugin will overwrite any data you have in the bucket name you set above if it already exists.
 
+* **Warning:** If the 'spa' option is set, the errorDocument setting will be ignored.
```

Comment:
If `spa: true` the errorDocument will be ignored.

See my above comment on related to the `spa` config value.